contracts/exchangeIssuance/ExchangeIssuanceLeveraged.sol

```diff
@@ -217,6 +214,65 @@ contract ExchangeIssuanceLeveraged is ReentrancyGuard, FlashLoanReceiverBaseV2{
         _approveToken(_token);
     }
 
+    /**
```

Comment:
Maybe we should mention somewhere in the comments that ther will be an error if the swap paths overlap. (In terms of exchange and path), the size of which depends on the liquidity.